src/chain/network_event.rs

```diff
@@ -78,6 +78,9 @@ pub enum AccumulatingEvent {
     NeighbourMerge(Digest256),
     SectionInfo(EldersInfo),
 
+    // Voted for received message with keys to we can update neighbour_info.
```

Comment:
Not quite clear of what this comment means.
does it mean `Voted for received message with keys that we can update our neighbour_info to` ?

Reply:
will fix